incent-config/Dangerfile

```diff
@@ -34,13 +34,43 @@ def multiple_program_sets?
   modified_program_sets.count > 1
 end
 
-def program_set
+def program_set_from_pr_title
   return 'na' if not_applicable_pr_title?
 
-  @program_set ||=
+  @program_set_from_pr_title ||=
     directories.find { |dir| github.pr_title.include?("[#{dir}]") }.to_s
 end
 
+def program_set_from_branch
+  return 'na' if not_applicable_branch_name?
```

Comment:
@johnallen3d I'm not entirely sure this is needed - but I added it _just in case_ a branch might be named, for example, `na/update-gitignore` (without `feature`).